examples/randomized-benchmark/two-qubit-rb/readme.md

```diff
@@ -0,0 +1,86 @@
+few generic inputs the user shall provide:
+- a  *QuantumMachinesManager* instance, which will be used to launch  the series of jobs necessary to play all synthezised sequences 
+- a *config* dictionary, used to open a Quantum Machine and to store the waveforms randomly generated
+- *N_Clifford*, the number of desired Clifford per sequence. Here an array of different values can be provided to retrieve different truncations
+- *N_sequences*: Number of random sequences to be generated for averaging
+- *two_qb_gate_macros*: Dictionary containing macros taking a baking object as input and using
+ baking play statements on necessary quantum elements to perform the three 2 qubit gates mentioned above ("CNOT", "SWAP" and "iSWAP" which are also keys of the dictionary).  The native two-qubit gate macro shall always contain a baking align statement before and after the playing statements
+- *single_qb_macros*: Optional dictionary in the case where a single qubit gate is not simply a QUA play statement on one quantum element, one can specify a dictionary of 
+a set of macros. Note that the dictionary shall then contain macros for all single qubit generators, i.e "I", "X", "Y", "X/2", "Y/2", "-X/2", "-Y/2".
+- *qubit_register*: Tuple containing names of two target qubits 
+- *seed*: Random seed
+
+
+# Run the experiment
+Before running the experiment, the user shall define all baking macros for computing Clifford generators for each of the two target qubits specified in the *qubit_register*.
+To help the user retrieve the correct quantum elements to be addressed when specifying one of the two qubits, we propose the implementation of the *Resolver* class, which aims to facilitate the retrieval of various quantum elements that might need to be involved for computing a specific quantum gate on the "logical qubits".
+One could in fact see a qubit as a set of quantum elements in the configuration, that do depend on the operation to be played on. Recall that within the *RBTwoQubits* class, the macros will be called with the following arguments: the baking object of reference (which shall always be 
+a parameter of the baking macros, as shown in the example), and the target qubit(s) (the two names given in the *qubit_register* argument of the class) on which the macros shall be applied. Single qubit gate macros only require one target qubit out of the two provided, whereas the two_qubit_gate macros require the two qubits of the register (a control and a target, always chosen to be in the same order for this experiment).
+
+The example shows briefly how one can run the experiment using the class. The method *run* uses one single argument,
+which is the main QUA program the user should define with its own measurement commands and its own stream_processing, such that the retrieval and fitting of results can be done directly. 
+The user shall build its QUA program according to two important guidelines:
```

Comment:
there is a very differnet writing style here than in other places in qua-libs. For example, Gal's writing style is much more informal. I suggest discussing with him about the tone

Reply:
True, but usually readmes of other examples are not as long and detailed as here, maybe should we add it as a tool as we did with one Qb RB in py-qua-tools?